bdai_ros2_wrappers/bdai_ros2_wrappers/service_client.py

```diff
@@ -1,63 +0,0 @@
-# Copyright (c) 2023 Boston Dynamics AI Institute Inc.  All rights reserved.
```

Comment:
Was this removed because now just doing `service.call` with a native client won't deadlock?

Reply:
Correct.